common/script-variable-generator/src/main/java/org/springframework/cloud/stream/module/common/ScriptModuleVariablesProperties.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.stream.module.common;
+
+import java.util.Properties;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.core.io.Resource;
+
+/**
+ * Configuration properties related to creating a script variables map.
+ *
+ * @author Eric Bottard
+ * @author Mark Fisher
+ */
+@ConfigurationProperties
+public class ScriptModuleVariablesProperties {
```

Comment:
I do like that better. In fact, I'm inclined to rename to just `ScriptVariableProperties`.